DependencyInjection/Configuration.php

```diff
@@ -19,7 +19,6 @@ public function getConfigTreeBuilder()
         $treeBuilder
             ->getRootNode()
             ->children()
-            ->scalarNode('default_class_namespace')->isRequired()->end()
```

Comment:
It might be better to mark this as deprecated (and optional) instead of outright removing it, just so that if and when a dev upgrades the client they'll be warned and told what needs to change. Otherwise the errors will be very cryptic.

Reply:
Sure, I've done that. It's not actually deprecated, its no longer used